app/Console/Commands/Init.php

```diff
@@ -148,19 +155,25 @@ protected function fillResources(): void
         $filePart = $this->loadReadmePart('RESOURCES.md');
 
         foreach (self::RESOURCES_ITEMS as $key => $title) {
-            if ($this->confirm("Are you going to use {$title}?", true)) {
-                $defaultLink = ($key === 'telescope') ? $this->appUrl . '/telescope' : '';
+            $defaultAnswer = ($key === 'telescope') ? $this->appUrl . '/telescope' : 'later';
```

Comment:
should we make the same logic for nova?

Reply:
@DenTray Sure, but step by step our command bacames more complex.

Reply:
@Goodmain I mean that for now you check only telescope key to generate default value with the app url, but actually we need to make the same logic for Nova as well